okta-hosted-login/README.md

```diff
@@ -27,7 +27,17 @@ You also need to gather the following information from the Okta Developer Consol
 
 - **Client ID** and **Client Secret** - These can be found on the "General" tab of the Web application that you created earlier in the Okta Developer Console.
 
-- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`.
+- **Issuer** - This is the URL of the [authorization server](https://developer.okta.com/docs/concepts/auth-servers/#available-authorization-server-types) that will perform authentication.
+
+This could either be an Org Authorization Server URL or Custom Authorization Server URL. 
+
+[Org Authorization Server](https://developer.okta.com/docs/concepts/auth-servers/#org-authorization-server) - Every Okta org would have this built-in authorization server. The issuer for this Authorization Server type would be `https://${yourOktaOrg}`. The `audience`, `claims`, `policies`, and `scope` OAuth 2.0 parameters of this authorization server cannot be customized. An access token that is minted by this authorization server type can only be consumed and validated by Okta. The access token cannot be used or validated by your own applications.
```

Comment:
In many of our blog posts, we use `https://{yourOktaOrg}` or https://{yourOktaDomain}`. We don't use the dollar sign.

Reply:
Noted, will update it in a new PR.

Reply:
#103